modules/ossm-integrating-distr-tracing-tempo.adoc

```diff
@@ -0,0 +1,90 @@
+////
+This module is included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-integrating-distr-tracing-tempo_{context}"]
+= Integrating distributed tracing platform (Tempo)
```

Comment:
I see TempoStack in our docs. Should it be...

```suggestion
= Integrating distributed tracing platform (TempoStack)
```

Consider this throughout

Reply:
The name of the product, according to the docs https://docs.openshift.com/container-platform/4.13/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.html its "Installing the distributed tracing platform (Tempo)"

Reply:
It's TempoStack when using it to reference "TempoStack instance," "TempoStack" resource, or "TempoStack" in the UI.